docs/apps/components/slider.md

```diff
@@ -0,0 +1,173 @@
+# Slider
+
+## Description
+
+ A **Slider** component allows users to visually select a numerical value within a predefined range.
+
+ It typically consists of a horizontal bar with a handle that users can drag to set their chosen value.
+
+<img src= "/apps/components/img/slider.png">
+
+## Use Case
+
+1. Setting price filters
+2. Choosing age brackets
```

Comment:
Simplify this to something like just Choosing age. I don't understand what age brackets are and I think some users might be confused too.